plugin.xml

```diff
@@ -80,7 +80,7 @@
     </config-file>
 
     <config-file target="AndroidManifest.xml" parent="/*">
-      <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" android:maxSdkVersion="32" />
+      <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" android:maxSdkVersion="28" />
```

Comment:
Note that I set this a few years ago because of android 13 issues. 28 is actually correct, though, as the permission was no longer required from sdk level 29